pandas/tests/base/test_unique.py

```diff
@@ -9,10 +14,20 @@
 from pandas.tests.base.common import allow_na_ops
 
 
+def maybe_perf_warn(using_pyarrow):
```

Comment:
ideally move to the _test_decorators.py (or similar) e.g. this is a general testing function.